src/README.md

```diff
@@ -1,9 +1,14 @@
+<<<<<<< HEAD
```

Comment:
When you merge files, you need to address it when this happens - I'm not sure how this one snuck through.

Reply:
I think you'll want to keep everything below the =====, not above in this case.

Reply:
It is weird. I already fixed it and it is not in my local README.md.